### ISSUE TITLE
[Feature]: Create OpenAPI specification and generate Zendesk connectors

### ISSUE THREAD

**Issue by kapilraajP** [ballerina-platform/ballerina-extended-library#44]
### Connector Name

connectors/openapi (Open API Connector)

### Suggested feature

As Zendesk APIs don't have existing OpenAPI specifications, We need to write OpenAPI specification and generate connectors using OpenAPI Tool. List of Zendesk APIs available and their connector availability in other vendors are denoted in the [document](https://docs.google.com/spreadsheets/d/1BZtaCARF_mIJiRcvw6jdzIeNEK0dNuwxZlpSjRb68IA/edit#gid=1794810240)

**Comment by LakshanSS** [comment]
@kapilraajP Shall we close this if it is done?

**Comment by kapilraajP** [comment]
Yes. Closed the issue. Since those two connector PRs are already merged
